packages/common-server/src/template.ts

```diff
@@ -121,6 +121,14 @@ class TemplateHelpers {
       const day = date.getDay();
       return day;
     },
+
+    capture: (text: string, pattern: string): string => {
```

Comment:
Let's also modify `genTrackPayload` in this file to get data for this helper method.